src/controllers/hawkerCentreController.ts

```diff
@@ -1,44 +1,44 @@
 import HawkerCentre from '../models/HawkerCentre';
 import {Request, Response, NextFunction} from 'express';
 
-async function postIdFormatting(req: Request, res: Response, next: NextFunction) {
-  try {
-    const hawkerCentre = {id: req.body['hawkerCentreId'], ...req.body};
-    delete hawkerCentre['hawkerCentreId'];
-    req.body = hawkerCentre;
-    next();
-  } catch (err) {
-    next(err);
-  }
-}
+// async function postIdFormatting(req: Request, res: Response, next: NextFunction) {
```

Comment:
I think it will be better to completely remove unused code to prevent cluttering. We can refer to the older commits should we need it again in the future.